lib/cli/src/commands/create_exe.rs

```diff
@@ -572,10 +618,6 @@ impl CreateExe {
                 cmd.arg("-lc");
             }
             cmd.arg("-lunwind");
-            cmd.arg("-OReleaseSafe");
```

Comment:
Is the default optimization level in Zig debug?
That would mean we are always doing debug builds?

Reply:
No, I should add this back